lib/uplinkc/object.go

```diff
@@ -95,11 +96,11 @@ type Upload struct {
 
 // upload uploads a new object, if authorized.
 //export upload
```

Comment:
nit: I've mostly seen `//export` being above the function doc comment, let's move this one to above in this same PR

Reply:
:+1: